docs/core/man/doveadm-import.1.md

```diff
@@ -43,8 +43,7 @@ level if *dest_parent* is empty (""). The *search_query* can be used to
 restrict which mailboxes or messages are imported. By default the import
 is done in destination user's context, you can use -U to change this.
 
-In the first form, [[man,doveadm]] will executed the **import** action
-mails will be imported for all users.
+In the first form, the **import** action will be performed for all users.
 
 In the second form, the mails will be imported for all users listed in
 the given *file*.
```

Comment:
I think would be better if these two first/second form sentences were synced. So I think better to use what is currently in the second sentence: "In the first form, the mails will be imported for all users."

Although I don't think it's very clear what the forms are even referring to, might as well clarify that by adding the parameter. Like "In the first form (-A parameter), ..."

Reply:
(same for 3rd and final forms)